README.md

```diff
@@ -10,6 +10,7 @@ Harmony service for subsetting L2 data. l2ss-py supports:
 - Spatial subsetting
 - Temporal subsetting
 - Variable subsetting
+- Shapefile subsetting
```

Comment:
Rather than adding a new item to this list I think we should do something like
> Spatial subsetting (via Bounding Box or Shapefile)

To make it more clear that it is still a spatial subset

Reply:
Just saw your other comments. Maybe we need to update documentation to specify GeoJSON is supported?